src/modules/flow/float/float.c

```diff
@@ -962,4 +962,36 @@ wave_generator_sinusoidal_open(struct sol_flow_node *node,
     return 0;
 }
 
+// =============================================================================
+// FPCLASSIFY
+// =============================================================================
+
+static int
+classify_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id, const struct sol_flow_packet *packet)
+{
+    double value;
+    int r;
+
+    r = sol_flow_packet_get_drange_value(packet, &value);
+    SOL_INT_CHECK(r, < 0, r);
+
+    switch (fpclassify(value)) {
+        case FP_NAN:
```

Comment:
Minor coding style issue: please don't indent `case` labels. When in doubt, use the `check-style.sh` script.